search_service/models/badge.py

```diff
@@ -0,0 +1,42 @@
+from .base import Base
+from typing import Set
+
+import attr
+from marshmallow_annotations.ext.attrs import AttrsSchema
+
+
+@attr.s(auto_attribs=True, kw_only=True)
```

Comment:
I went this route per Tamika's comments on the frontendlibrary pr. I don't love it, would love to hear feedback on what might be preferred. Either way, if we're moving this part of the data model into the search service, we need some sort of class. It doesn't seem like it SHOULD inherit from the `Base` class but I was following the existing pattern.